dll/win32/kernel32/client/sysinfo.c

```diff
@@ -1,14 +1,14 @@
 /*
  * PROJECT:         ReactOS Win32 Base API
- * LICENSE:         See COPYING in the top level directory
- * FILE:            dll/win32/kernel32/client/sysinfo.c
+ * LICENSE:         GPL-2.0-or-later (https://spdx.org/licenses/GPL-2.0-or-later)
  * PURPOSE:         System Information Functions
- * PROGRAMMERS:     Emanuele Aliberti
+ * COPYRIGHT:       Emanuele Aliberti
  *                  Christoph von Wittich
  *                  Thomas Weidenmueller
  *                  Gunnar Andre Dalsnes
  *                  Stanislav Motylkov (x86corez@gmail.com)
  *                  Mark Jansen (mark.jansen@reactos.org)
+ *                  Copyright 2023 Ratin Gao <ratin@knsoft.org>
```

Comment:
when adding yourself to this already long list, I would recommend to strip 2 inactive "Thomas Weidenmueller" and "Gunnar Andre Dalsnes" in return. (Alternatively don't extend the list even further).